dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/YesNoNaComponent.kt

```diff
@@ -36,10 +40,18 @@ class YesNoNaComponent(
     }
 
     override fun generateDefaultUploadConfig(uploadCategoryBuilder: UploadCategoryBuilder) {
-        requireDocumentSupportIn(setOf(NoDocumentSupport))
+        requireDocumentSupportIn(setOf(NoDocumentSupport, SimpleDocumentSupport))
+        val uploadComponentNameToUse = when (documentSupport) {
+            is NoDocumentSupport -> "YesNoNaFormField"
+            is SimpleDocumentSupport -> "YesNoNaBaseDataPointFormField"
```

Comment:
Documentsupport was not added in the viewconfig generation, it will only show Yes or No and no download link

Reply:
Probably the same for the decimal component

Reply:
actually there should be no restriction at all if no document support is specified at all (no "required" line present), but i added it nevertheless

Reply:
Nevermind. I misread something oversaw the `documentSupport` in `ocumentSupport.getFrameworkDisplayValueLambda(`.

Reply:
The addition of requireDocumentSupportIn with only these two parameters would break the dev tools output for sfdr , I think.

Reply:
why would that break the sfdr output?

Reply:
<img width="275" alt="Capture" src="https://github.com/d-fine/Dataland/assets/113936582/fbe0b62a-3673-4b89-b1dc-d8321d76e44e">
Da sollte noch "Yes" stehen eigentlich, oder? In der Api Response ist es drin.

Reply:
Leaving this open as a reminder for me to check it again